src/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>HealthriCatalogApp</title>
+  <title>gdiCatalogApp</title>
```

Comment:
What do you think of 'GDI - User Portal'?

Reply:
fixed